src/com/jalasoft/search/view/Window.java

```diff
@@ -0,0 +1,90 @@
+import javax.swing.JButton;
```

Comment:
Alvarex, lets use the Paolo's suggestion for copyrights.

/*
 * Search .java
 *
 * Copyright (c) 2018 Jalasoft.
 *
 * This software is the confidential and proprietary information of Jalasoft.
 * ("Confidential Information").  You shall not
 * disclose such Confidential Information and shall use it only in
 * accordance with the terms of the license agreement you entered into
 * with Jalasoft.
 */